app/models/group.rb

```diff
@@ -0,0 +1,7 @@
+class Group < ActiveRecord::Base
+  has_many :user_group_memberships
+  has_many :users, through: :user_group_memberships
+  belongs_to :moderator, class_name: :User, foreign_key: :moderator_id
```

Comment:
@Enriikke what do you think of this? I really like the idea of an attribute that explicitly says `moderator_id` as opposed to `user_id` which could maybe be misleading. Would you do this differently?